common/lib/xmodule/xmodule/tests/test_bulk_assertions.py

```diff
@@ -55,11 +59,22 @@ class TestBulkAssertionTestCase(BulkAssertionTest):
 
     # pylint: disable=bad-super-call
 
+    def _is_arg_in_assertion_methods_dict(self, argument):
+        """
+        Takes in an argument, and returns whether
+        """
+        return type(argument) == str and argument in ASSERTION_METHODS_DICT
+
     def _run_assertion(self, assertion_tuple):
         """
         Run the supplied tuple of (assertion, *args) as a method on this class.
         """
         assertion, args = assertion_tuple[0], assertion_tuple[1:]
+        args_list = list(args)
+        for count, argument in enumerate(args_list):
```

Comment:
nit: I'd go with `index` rather than `count` here.  To me at least, `count` implies the overall number of items in the list.